server/command.go

```diff
@@ -219,21 +239,60 @@ func (p *Plugin) runDisconnectCommand(user *model.User) (string, error) {
 }
 
 // runHelpCommand runs command to display help text.
-func (p *Plugin) runHelpCommand(user *model.User) (string, error) {
-	text := "###### Mattermost Zoom Plugin - Slash Command Help\n"
-	text += strings.ReplaceAll(helpText, "|", "`")
-
-	if p.canConnect(user) {
+func (p *Plugin) runHelpCommand() (string, error) {
+	text := starterText
+	text += strings.ReplaceAll(helpText+settingHelpText, "|", "`")
+	if p.configuration.EnableOAuth {
 		text += "\n" + strings.ReplaceAll(oAuthHelpText, "|", "`")
 	}
+
 	return text, nil
 }
 
+// run "/zoom setting" command, e.g: /zoom setting use_pmi true
+func (p *Plugin) runSettingCommand(settingCommands []string, user *model.User) (string, error) {
+	settingAction := ""
+	if len(settingCommands) > 0 {
+		settingAction = settingCommands[0]
+	}
+	switch settingAction {
+	case "use_pmi":
+		// here process the use_pmi command
+		if len(settingCommands) > 1 {
+			return p.runPMISettingCommand(settingCommands[1], user)
+		}
+		return "Set PMI option to \"true\"|\"false\"|\"ask\"", nil
+	case "":
+		return strings.ReplaceAll(starterText+settingPMIHelpText, "|", "`"), nil
+	default:
+		return fmt.Sprintf("Unknown Action %v", settingAction), nil
+	}
+}
+
+func (p *Plugin) runPMISettingCommand(usePMIValue string, user *model.User) (string, error) {
+	switch usePMIValue {
+	case trueString, falseString, zoomPMISettingValueAsk:
+		if appError := p.API.UpdatePreferencesForUser(user.Id, []model.Preference{
+			{
+				UserId:   user.Id,
+				Category: zoomPreferenceCategory,
+				Name:     zoomPMISettingName,
+				Value:    usePMIValue,
+			},
+		}); appError != nil {
+			return "Cannot update preference in zoom setting", nil
```

Comment:
make a constant for this error and first letter should be small do both of these everywhere